OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/DiscussionCommentsAdapter.java

```diff
@@ -111,6 +115,11 @@ public void onBindViewHolder(RecyclerView.ViewHolder viewHolder, int position) {
         final ResponseOrCommentViewHolder holder = (ResponseOrCommentViewHolder) viewHolder;
         final DiscussionComment discussionComment;
         final IconDrawable iconDrawable;
+        final int viewPadding = (int) context.getResources().getDimension(R.dimen.discussion_responses_box_padding);
+        final View commentRow = holder.discussionCommentRow;
+
+        commentRow.setPadding(commentRow.getPaddingLeft(), commentRow.getPaddingTop(),
+                commentRow.getPaddingRight(), viewPadding);
```

Comment:
Hmm, it seems very clunky to have the padding hardcoded dynamically in the adapter code itself. How about putting the view inside a container (e.g. `FrameLayout`) that set's it's own minimum height same as the amount of padding defined by the view, so that the view can have the padding to increase it's clickable area, but hiding it would not remove all the padding from the layout either?